dataland-frontend/src/components/resources/frameworkDataSearch/TwoLayerDataTable.vue

```diff
@@ -48,71 +48,75 @@
               slotProps.data.content[reportingPeriodWithDataId.dataId] !== null
             "
           >
-            <template v-if="Array.isArray(slotProps.data.content[reportingPeriodWithDataId.dataId])">
-              <a
-                v-if="
-                  slotProps.data.content[reportingPeriodWithDataId.dataId].length > 1 ||
-                  slotProps.data.content[reportingPeriodWithDataId.dataId].some((el) => typeof el === 'object')
-                "
-                @click="
-                  openModalAndDisplayValuesInSubTable(
-                    slotProps.data.content[reportingPeriodWithDataId.dataId],
-                    slotProps.data.kpiLabel,
-                    slotProps.data.kpiKey,
-                    slotProps.data.kpiFormFieldComponent,
-                  )
-                "
-                class="link"
-                >Show "{{ slotProps.data.kpiLabel }}"
-                <em class="material-icons" aria-hidden="true" title=""> dataset </em>
-              </a>
-
-              <span v-else> {{ slotProps.data.content[reportingPeriodWithDataId.dataId][0] }} </span>
-            </template>
-            <span
-              v-else-if="
-                slotProps.data.kpiFormFieldComponent === 'PercentageFormField' &&
-                slotProps.data.content[reportingPeriodWithDataId.dataId] !== ''
-              "
-            >
-              {{ slotProps.data.content[reportingPeriodWithDataId.dataId] }} %</span
-            >
             <span
-              v-else-if="
-                typeof slotProps.data.content[reportingPeriodWithDataId.dataId] === 'object' &&
-                slotProps.data.content[reportingPeriodWithDataId.dataId]?.value !== undefined
-              "
+              :data-test="`${reportingPeriodWithDataId.reportingPeriod}_${slotProps.data.categoryKey}_${slotProps.data.kpiKey}`"
```

Comment:
I can confirm that the error described in the ticket is gone. However, I am not sure how this change fixes it. If I see it correctly there was only a span added that contains only data-test information. How does this lead to resolving the bug?

Reply:
It is only needed for the test. Adding a span in this spot allows us to wrap every value created within the conditionals nested within. Otherwise we'd need to wrap each value with their own span within each conditional.
I added this span to improve  targeting of specific values within tests. It was possible before, yes, but rather difficult and cumbersome code. This way we can directly target each cell easily.